src/main.c

```diff
@@ -46,9 +46,21 @@ int main(void) {
     cdc_uart_init();
     tusb_init();
     probe_init();
-    led_init();
+    led_init(); 
 
-    picoprobe_info("Welcome to Picoprobe!\n");
+/*
+ * Keep led_init() at the end, so that the time it takes to count an initial
+ * value down to zero can be checked with a 'scope or logic analyser.
+ */
+
+    picoprobe_info("Welcome to Picoprobe! ");
+#ifdef PROBE_PIN_RESET
+    picoprobe_info("Reset: GP%d ", PROBE_PIN_RESET);
+#endif
+#ifdef PICOPROBE_LED
+    picoprobe_info("LED: GP%d", PICOPROBE_LED);
+#endif
```

Comment:
Should this have a similar section for reporting the `PICOPROBE_WS2812` value?

Reply:
I added WS8212 support later, and decided not to do a pull for it since it's not standard on the Pico board.

If anybody does want it please feel free to take it directly from https://github.com/MarkMLl/picoprobe_led. I must stress at this point that despite being an "old hand" as far as embedded systems etc. are concerned I'm very much a beginner with cmake and git, and even less confident with them than I am in my ability to get C code right.

The major thing I've tried to do in the patch is to get the timing sorted out, so that there was some sort of visible feedback that data was being transferred. At present it's probably excessively sensitive, but with a WS8212 (Waveshare RP2040-Zero board) it's just about possible to see that multiple things are happening.

Reply:
> I added WS8212 support later, and decided not to do a pull for it since it's not standard on the Pico board.
If anybody does want it...

Huh, but it's included in this PR? Maybe you created this PR from the wrong branch in your repo or something?